arborist/server/router_model.go

```diff
@@ -0,0 +1,60 @@
+// This file defines addPolicyRouter for adding a router for the set of
+// endpoints under `/policy` to a main router. This router handles list,
+// create, read, update, and delete operations on the policies in the arborist
+// engine.
+
+package server
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	"github.com/gorilla/mux"
+
+	"github.com/uc-cdis/arborist/arborist"
+)
+
+func handleForceUpdateModel(engine *arborist.Engine) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		body, err := ioutil.ReadAll(r.Body)
+		if err != nil {
+			writeJSONReadError(w, err)
+			return
+		}
+
+		response := engine.HandleForceUpdateModelFromS3(body)
+
+		err = response.Write(w, wantPrettyJSON(r))
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+			return
+		}
+	})
+}
+
+func handleUploadModelToS3(engine *arborist.Engine) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		body, err := ioutil.ReadAll(r.Body)
+		if err != nil {
+			writeJSONReadError(w, err)
+			return
+		}
+
+		response := engine.HandlePostModelToS3(body)
+
+		err = response.Write(w, wantPrettyJSON(r))
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+			return
+		}
+
+	})
+}
+
+// syncModelRouter attaches the handlers defined in this file to a main router,
+// using the prefix `/model`.
+func (server *Server) syncModelRouter(mainRouter *mux.Router) {
```

Comment:
can we keep the naming consistent with the other ones and name this `addModelRouter`?

Reply:
Fixed in bc25eb4ad8ccd44d86b7a24694e3989552e789c7